pdfmaker/css/_modules/base.scss

```diff
@@ -1111,6 +1111,24 @@ p[class*="ExtractSource"] {
   orphans: 2;
 }
 
+p[class*="Extract-Noteextn"] {
```

Comment:
Note paragraphs will actually automatically get wrapped in a blockquote (see https://github.com/macmillanpublishers/bookmaker/blob/master/core/htmlmaker/wordtohtml.xsl#L310), so you only need to add definitions here that should be applied on top of whatever already gets applied to the blockquote (e.g., margins will be handled by the blockquote, but text-align won't).